bears/scss/SCSSLintBear.py

```diff
@@ -21,4 +23,195 @@ class SCSSLintBear:
 
     @staticmethod
     def create_arguments(filename, file, config_file):
-        return filename,
+        return filename, '--config', config_file
+
+    @staticmethod
+    def generate_config(filename, file,
+                        check_bang_format: bool=True,
+                        space_before_bang: bool=True,
+                        space_after_bang: bool=False,
+                        allow_chained_classes: bool=False,
+                        allow_color_keywords: bool=False,
+                        use_color_variables: bool=True,
+                        allow_debug_statement: bool=False,
+                        check_declaration_order: bool=True,
+                        allow_duplicate_properties: bool=False,
+                        allow_consecutives_duplicate_property: bool=False,
+                        else_placement_style: str='same_line',
+                        force_empty_line_between_blocks: bool=True,
+                        allow_empty_rules: bool=False,
+                        hexadecimal_length_style: str='short',
+                        hexadecimal_notation_case: str='lowercase',
+                        allow_hexadecimal_validation: bool=True,
+                        allow_id_selector: bool=False,
+                        allow_important_rule_in_properties: bool=False,
+                        use_spaces: bool=True,
+                        indent_size: int=2,
+                        leading_zero: str='exclude_zero',
+                        allow_mergeable_selectors: bool=False,
+                        allow_leading_underscore: bool=True,
+                        function_name_case: str='hyphenated_lowercase',
```

Comment:
I'd allow the user to write `hyphenated lowercase` or so so it's nicer to read, so you just do a replacement from spaces to underscores in the bear for him :)